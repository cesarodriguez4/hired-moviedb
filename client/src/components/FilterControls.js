import PropTypes from 'prop-types';

export const FilterControls = ({releaseFilterUp, onReleaseFilterChange}) => {
    return <div>
        Filter by: 
        <div>
            Release date:
            <span className="ml-2 cursor-pointer" onClick={onReleaseFilterChange}>
                {releaseFilterUp ? '⬆️' : '⬇️'}
            </span>
        </div>
    </div>
}

FilterControls.propTypes = {
    releaseFilterUp: PropTypes.bool.isRequired,
    onReleaseFilterChange: PropTypes.func.isRequired
};
