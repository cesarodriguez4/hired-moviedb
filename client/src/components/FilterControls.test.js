import {render} from '@testing-library/react';
import '@testing-library/jest-dom';
import { FilterControls } from './FilterControls';

test('renders filter controls component', () => {
  const {container} = render(<FilterControls />);
  expect(container).toMatchSnapshot();
});