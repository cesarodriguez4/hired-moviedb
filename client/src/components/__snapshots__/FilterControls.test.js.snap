// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders filter controls component 1`] = `
<div>
  <div>
    Filter by:
    <div>
      Release date:
      <span
        class="ml-2 cursor-pointer"
      >
        ⬇️
      </span>
    </div>
  </div>
</div>
`;
